app/models/redirection.rb

```diff
@@ -15,4 +17,11 @@ def next_url
   def previous_url
     previous.url
   end
+
+  def tag_uri
+    unchanging_hostname = "hotlinewebring.club"
```

Comment:
Sorry, I should have put this in the PR description instead of just the commit message, but this shouldn't ever change, even if the domain does:

>According to the [atom standard](http://tools.ietf.org/html/rfc4287#page-20) entries in an atom feed should never, ever change. They should also not be the same as the full URI (or really even be a URI) to limit confusion as to what they are used for. They are a unique string used for identification.

>`hotlinewebring.club` is hardcoded into for tag uri formula so that even if the app's domain changes, the unique, never changing ids will stay the same. Hopefully.

> Here is a blog post for reference: https://web.archive.org/web/20081225232147/http://diveintomark.org/archi
ves/2004/05/28/howto-atom-id